guides/source/getting_started.md

```diff
@@ -1115,6 +1115,48 @@ Let's refactor this to use these helpers:
 </div>
 ```
 
+### Displaying Flash Messages
+
+The
+[Flash](https://edgeapi.rubyonrails.org/classes/ActionDispatch/Flash.html)
```

Comment:
I know this is how it's written in the docs, but "the Flash" sounds weird to me. Maybe 
```suggestion
The
[`flash` object](https://edgeapi.rubyonrails.org/classes/ActionDispatch/Flash.html)
```

Reply:
I guess we keep it then?